docs/content/en/docs/getting-started/observability/_index.md

```diff
@@ -20,165 +35,61 @@ The observability data is an amalgamation of the following:
-
-The Keptn Lifecycle Toolkit sits in the scheduler
-so it can trace all activities of all deployment workloads on the cluster,
-no matter what tool is used for the deployment.
-This same mechanism allows KLT to inject pre- and post-deployment checks
-into all deployment workloads;
-we discuss this in another exercise.
-
-KLT uses metadata to identify the workloads of interest.
-To integrate KLT with your applications,
-you need to populate the metadata it needs.
-This requires the following steps:
-
-- Define a Keptn application
-- Annotate the `Deployment` resource to recognize your Keptn application
-
-### Define the Keptn application
-
-A Keptn application defines the workloads
-to be included in your Keptn Application.
-We will use the application discovery feature
-to automatically generate a Keptn Application
-that includes all workloads on the cluster,
-regardless of the tools being used.
-
-A Keptn application aggregates multiple workloads
-that belong to a logical app into a single
-[KeptnApp](../../yaml-crd-ref/app.md)
-resource.
-
-You can view a sample of this file in the
-[keptn-app.yaml](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/keptn-app.yaml.tmp)
-file.
-You see the metadata that names this `KeptnApp`
-and identifies the namespace where it lives:
-
-```yaml
-metadata:
-  name: simpleapp
-  namespace: simplenode-dev
-```
-
-You can also see the `spec.workloads` list.
-In this simple example,
-we only have one workload defined
-but most production apps will have multiple workloads defined.
-
-You can create the YAML file to define the resource manually
-but the easier approach is to let KLT create this definition for you.
-This requires that you annotate all your workloads
-(`Deployments`, `Pods`, `StatefulSets`, `DaemonSets`, and `ReplicaSets`
-as described in
-[Use Keptn automatic app discovery](../../implementing/integrate/#use-keptn-automatic-app-discovery).
-
-### Annotate your Deployment resource
-
-Follow the instructions in
-[Annotate workload](../../implementing/integrate/#basic-annotations)
-to apply basic annotations to your `Deployment` resource.
-
-The
-[simplenode-dev-deployment.yaml](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/simplenode-dev-deployment.yaml/)
-file defines the `Deployment` resource for our example.
-You see that the `metadata` specifies the same
-`name` and `namespace` values defined in the `KeptnApp` resource.
-
-The example file also includes annotations for
-pre- and post-deployment activities.
-We will discuss those in a separate exercise.
+   [Integrate KLT with your applications](../../implementing/integrate)
+   to integrate KLT with your Kubernetes cluster:
+
+   - Follow the instructions in
+     [Annotate workload](../../implementing/integrate/#basic-annotations)
+     to integrate the Lifecycle Toolkit into your Kubernetes cluster
+     by applying basic annotations to your `Deployment` resource.
```

Comment:
I would suggest to use the more generic "Workload" instead of "Deployment" here, as we also support StatefulSets and DaemonSets in addition to Deployments

Reply:
or maybe: ... by applying basic annotations to your Workload (i.e., Deployment, DaemonSet, StatefulSet) resource.

Reply:
But the "basic annotations" being discussed here are actually applied to the `Deployment` resource/CRD/ .   But this uncovers a big problem that needs some work but separate from this PR.  See https://github.com/keptn/lifecycle-toolkit/issues/1451 .

Reply:
ok that makes sense, thanks for clarifying